src/components/structures/auth/SetupEncryptionBody.tsx

```diff
@@ -89,7 +89,7 @@ export default class SetupEncryptionBody extends React.Component<IProps, IState>
     private onVerifyClick = (): void => {
         const cli = MatrixClientPeg.get();
         const userId = cli.getSafeUserId();
-        const requestPromise = cli.requestVerification(userId);
+        const requestPromise = cli.getCrypto()!.requestOwnUserVerification();
```

Comment:
> So we assume that the crypto is defined here ?

Yes. I'm reasonably sure that `SetupEncryptionBody` is only used when `getCrypto` returns something -- and `SetupEncryptionStore` (which is required for `SetupEncryptionBody`) makes the same assumption.

> Can we use instead cli.getCrypto()?.requestOwnUserVerification() ?

mmm; unfortunately I don't think it's as simple as that. `VerificationRequestDialog` requires a (non-null) `verificationRequestPromise`; we could add an `if` statement and return instead of showing the dialog, but then we get into untestable code - and if the code is ever reachable then the UX is arguably just as bad as spamming an exception into the logs.